src/main/java/io/javarig/util/Utils.java

```diff
@@ -22,4 +24,10 @@ public static Field getOwnOrInheritedFieldByName(Class<?> type, String fieldName
     public static boolean isClassGeneric(Class<?> aClass) {
         return aClass != null && aClass.getTypeParameters().length > 0;
     }
+    public static String removeUnsupportedRegexCharacters(String regexPattern){// if other chars are unsupported maybe convert to list and loop
```

Comment:
why these characters are unsupported

Reply:
limitation of the used library ( it is the only one i found that accepts min max size as arguments, so if we want to use another library we will not be able to accept a combination of size constraint and regex)

Reply:
Maybe we drop the size constraint when the pattern is given ? If the user wants a pattern with a size, he can specify it in the pattern .

Reply:
@HamzaBenyazid done